app/controllers/registrations_controller.rb

```diff
@@ -47,7 +47,7 @@ def account_update_params
   end
 
   def after_sign_up_path_for(resource)
-    adoptable_pets_path
+    new_submission_path
```

Comment:
This means that right after supplying email and password etc, the user will be shown the organization's form.

Reply:
Maybe something to consider later on, but if the user has not yet completed and saved the form, the after sign in path should take them to the form. Otherwise, if form is completed and saved, after sign in path could be the adopter or foster dashboard.